pkg/azure/aks/price_store.go

```diff
@@ -16,13 +16,40 @@ import (
 const (
 	AzurePriceSearchFilter = `serviceName eq 'Virtual Machines' and priceType eq 'Consumption'`
 	AzureMeterRegion       = `'primary'`
+
+	DefaultInstanceFamily = "General purpose"
```

Comment:
Ah this is interesting. TIL if you have a space between consts, `go-fmt` won't line up the equals signs. I think the intent here is to group consts?

Reply:
updated to remove line between, and `go-fmt` lines up the equals sign!